corehq/apps/callcenter/views.py

```diff
@@ -40,6 +41,7 @@ def static_options(self):
         return []
 
 
+@location_safe
```

Comment:
This view is used by a select2 widget much like the `EmwfOptionsView` from which it inherits is used by a select2 widget to display a combined list of users, groups, and locations.

If a domain has the call center enabled, a default case owner for call center cases can be selected. This configuration happens on the `/a/<domain>/settings/project/basic/` page, and the user does that selection with the widget introduced in this PR.
A few lines below, in the `get_locations_query()` method of this class, `SQLLocation` objects are filtered with the `accessible_to_user()` method. My understanding is that that method is whats needed to make a view "location safe"?